scio-bigquery/src/main/scala/com/spotify/scio/bigquery/dynamic/syntax/SCollectionSyntax.scala

```diff
@@ -57,13 +80,20 @@ final class DynamicBigQueryOps[T](private val self: SCollection[T]) extends AnyV
         .write()
         .to(destinations)
         .withFormatFunction(Functions.serializableFn(formatFn))
+        .withExtendedErrorInfo()
```

Comment:
If we set `withExtendedErrorInfo()` here, does it matter which `ExtendedErrorInfo` is actually passed in for `extendedErrorInfo` param?

Wouldn't it defeat the purpose of subsequent pattern match on `extendedErrorInfo: ExtendedErrorInfo`? _(Line 92-95)_:
```
transform = extendedErrorInfo match {
  case Disabled => transform
  case Enabled  => transform.withExtendedErrorInfo()
}
```

Was this an artifact of incorrect merge conflict resolution?

Reply:
Hi @byungk Nice catch!  not sure what really happen but probably was what you mentioned.